src/gui/src/layoutViewer.h

```diff
@@ -211,9 +219,11 @@ class LayoutViewer : public QWidget, public odb::dbBlockCallBackObj
   void drawRulers(Painter& painter);
   Selected selectAtPoint(odb::Point pt_dbu);
 
-  odb::Rect screenToDBU(const QRect& rect);
-  odb::Point screenToDBU(const QPoint& point);
-  QRectF dbuToScreen(const odb::Rect& dbu_rect);
+  void zoom(const odb::Point& focus, qreal factor, bool do_delta_focus);
+
+  qreal computePixelsPerDBU(const QSize& size, const odb::Rect& dbu_rect);
+  odb::Rect getPaddedRect(const odb::Rect& rect, double factor = 0.05);
+  void setCenteringShift();
```

Comment:
This name isn't obvious as to its function and it is strange for a setter to take no arguments.  Maybe

// Compute and store the offset necessary to center the block in the viewport.
void computeCenteringOffset();